src/server/pps/server/api_server.go

```diff
@@ -1337,16 +1345,6 @@ func (a *apiServer) workerServiceIP(ctx context.Context, deploymentName string)
 func (a *apiServer) pipelineManager(ctx context.Context, pipelineInfo *pps.PipelineInfo) {
 	// Clean up workers if the pipeline gets cancelled
 	pipelineName := pipelineInfo.Pipeline.Name
-	go func() {
-		// Clean up workers if the pipeline gets cancelled
-		<-ctx.Done()
-		rcName := PipelineRcName(pipelineInfo.Pipeline.Name, pipelineInfo.Version)
-		if err := a.deleteWorkers(rcName); err != nil {
-			protolion.Errorf("error deleting workers for pipeline: %v", pipelineName)
-		}
-		protolion.Infof("deleted workers for pipeline: %v", pipelineName)
-	}()
-
```

Comment:
It seems like there's probably a matching call to create the workers that might error now because the workers already exist. Is that true and if so do we trap that error in some way?